src/test/java/uk/gov/hmcts/reform/sscs/service/LetterUtilsTest.java

```diff
@@ -216,6 +212,121 @@ public void isAlternativeLetterFormatRequired(SubscriptionType subscriptionType)
         assertTrue(LetterUtils.isAlternativeLetterFormatRequired(wrapper, new SubscriptionWithType(EMPTY_SUBSCRIPTION, subscriptionType)));
     }
 
+    @Test
+    public void givenAnOtherParty_thenIsAlternativeLetterFormatRequired() {
+        List<CcdValue<OtherParty>> otherPartyList = new ArrayList<>();
+        CcdValue<OtherParty> ccdValue = CcdValue.<OtherParty>builder().value(OtherParty.builder()
+                .id("1")
+                .reasonableAdjustment(ReasonableAdjustmentDetails.builder().wantsReasonableAdjustment(YesNo.YES).build())
+                .build()).build();
+        otherPartyList.add(ccdValue);
+
+        SscsCaseData caseData = SscsCaseData.builder()
+                .otherParties(otherPartyList).build();
+        SscsCaseDataWrapper caseDataWrapper = SscsCaseDataWrapper.builder()
+                .newSscsCaseData(caseData)
+                .oldSscsCaseData(caseData)
+                .notificationEventType(APPEAL_RECEIVED_NOTIFICATION)
+                .build();
+        NotificationWrapper wrapper = new CcdNotificationWrapper(caseDataWrapper);
+        SubscriptionWithType subscriptionWithType = new SubscriptionWithType(EMPTY_SUBSCRIPTION, OTHER_PARTY);
+        subscriptionWithType.setPartyId(1);
+        assertTrue(LetterUtils.isAlternativeLetterFormatRequired(wrapper, subscriptionWithType));
+    }
+
+    @Test
+    public void givenAnOtherPartyWithAppointeeThatWantsReasonableAdjustment_thenIsAlternativeLetterFormatRequiredForAppointee() {
+        List<CcdValue<OtherParty>> otherPartyList = new ArrayList<>();
+        CcdValue<OtherParty> ccdValue = CcdValue.<OtherParty>builder().value(OtherParty.builder()
+                .id("1")
+                .appointee(Appointee.builder().id("2").build())
+                .isAppointee("Yes")
+                .reasonableAdjustment(ReasonableAdjustmentDetails.builder().wantsReasonableAdjustment(YesNo.NO).build())
+                .appointeeReasonableAdjustment(ReasonableAdjustmentDetails.builder().wantsReasonableAdjustment(YesNo.YES).build())
+                .build()).build();
+        otherPartyList.add(ccdValue);
+
+        SscsCaseData caseData = SscsCaseData.builder()
+                .otherParties(otherPartyList).build();
+        SscsCaseDataWrapper caseDataWrapper = SscsCaseDataWrapper.builder()
+                .newSscsCaseData(caseData)
+                .oldSscsCaseData(caseData)
+                .notificationEventType(APPEAL_RECEIVED_NOTIFICATION)
+                .build();
+        NotificationWrapper wrapper = new CcdNotificationWrapper(caseDataWrapper);
+        SubscriptionWithType subscriptionWithType = new SubscriptionWithType(EMPTY_SUBSCRIPTION, OTHER_PARTY);
+        subscriptionWithType.setPartyId(2);
+        assertTrue(LetterUtils.isAlternativeLetterFormatRequired(wrapper, subscriptionWithType));
+    }
+
+    @Test
+    public void givenAnOtherPartyWithRepThatWantsReasonableAdjustment_thenIsAlternativeLetterFormatRequiredForAppointee() {
```

Comment:
Change name of the test, instead of Appointee, maybe use OtherPartyRep

Reply:
Good spot, updated